### PR TITLE
Privacy Sandbox new FAQ - Does SameSite become irrelevant after the third-party cookie deprecation?

### PR DESCRIPTION
**Pull request by antoinebisch**
Added FAQ: Does `SameSite` become irrelevant after the third-party cookie deprecation?

Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Review by samdutton — APPROVED** [review]
LGTM with minor tweak.

Assume this is OK with (or written by) @rowan-m.

**Comment by heyawhite** [comment]
@samdutton @antoinebisch @rowan-m should this still be reviewed and published?

**Comment by samdutton** [comment]
> @samdutton @antoinebisch @rowan-m should this still be reviewed and published?

Definitely one for @rowan-m 💵 🏈

File: site/en/docs/privacy-sandbox/faq/index.md [hunk 1]
```diff
@@ -141,6 +141,13 @@ For more detailed information, see
 
 Chrome on iOS and iPadOS does not support Chrome origin trials.
 
+### Does `SameSite` become irrelevant after the third-party cookie deprecation?
+
+ - `SameSite=Lax` is the current default, so while it does not strictly *need* to be included it's good practice to specify it for cross-browser consistency.
+ - `SameSite=Strict` continues to be a more restrictive option, for cookies that must only be sent when the user is already on the site. This is and remains a good security practice for cookies that are part of managing particularly sensitive access.
+ - `SameSite=None` should continue to be sent for cross-browser consistency. However, Chrome's proposed change to phase out third-party cookies would result in those cookies no longer being sent in cross-site contexts.
+ - The exception to this is cookies that are modified by either the `Partitioned` or `SameParty` attributes, which allows for a subset of cross-site use cases. *Note:* both these attributes are only proposals at this stage and the final format may be different.
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/faq/index.md:149]:
```suggestion
 - The exception to this is cookies that are modified by either the [CHIPS](https://developer.chrome.com/docs/privacy-sandbox/chips/) or [First-Party Sets](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/) proposals, which allows for a subset of cross-site use cases. *Note:* these proposals are under active discussion, so final formats and functionality may change.
```

File: site/en/docs/privacy-sandbox/faq/index.md [hunk 2]
```diff
@@ -141,6 +141,13 @@ For more detailed information, see
 
 Chrome on iOS and iPadOS does not support Chrome origin trials.
 
+### Does `SameSite` become irrelevant after the third-party cookie deprecation?
+
+ - `SameSite=Lax` is the current default, so while it does not strictly *need* to be included it's good practice to specify it for cross-browser consistency.
+ - `SameSite=Strict` continues to be a more restrictive option, for cookies that must only be sent when the user is already on the site. This is and remains a good security practice for cookies that are part of managing particularly sensitive access.
+ - `SameSite=None` should continue to be sent for cross-browser consistency. However, Chrome's proposed change to phase out third-party cookies would result in those cookies no longer being sent in cross-site contexts.
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/faq/index.md:148]:
```suggestion
 - `SameSite=None` should continue to be sent for cross-browser consistency. However, Chrome's proposed change to phase out third-party cookies would result in those cookies no longer being sent as is in cross-site contexts.
```

**Review by rowan-m — CHANGES_REQUESTED** [review]
Minor changes to bring this up to date, otherwise LGTM.

**Review by mihajlija — APPROVED** [review]

File: site/en/docs/privacy-sandbox/faq/index.md [hunk 3]
```diff
@@ -141,6 +141,13 @@ For more detailed information, see
 
 Chrome on iOS and iPadOS does not support Chrome origin trials.
 
+### Does `SameSite` become irrelevant after the third-party cookie deprecation?
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/faq/index.md:144]:
```suggestion
### Will `SameSite` become irrelevant after third-party cookies are deprecated?
```

File: site/en/docs/privacy-sandbox/faq/index.md [hunk 4]
```diff
@@ -141,6 +141,13 @@ For more detailed information, see
...
+
+ - `SameSite=Lax` is the current default, so while it does not strictly *need* to be included it's good practice to specify it for cross-browser consistency.
+ - `SameSite=Strict` continues to be a more restrictive option, for cookies that must only be sent when the user is already on the site. This is and remains a good security practice for cookies that are part of managing particularly sensitive access.
+ - `SameSite=None` should continue to be sent for cross-browser consistency. However, Chrome's proposed change to phase out third-party cookies would result in those cookies no longer being sent as is in cross-site contexts.
+ - The exception to this is cookies that are modified by either the [CHIPS](https://developer.chrome.com/docs/privacy-sandbox/chips/) or [First-Party Sets](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/) proposals, which allows for a subset of cross-site use cases. *Note:* these proposals are under active discussion, so final formats and functionality may change.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/faq/index.md:149]:
```suggestion
 - `SameSite=Lax` is the current default. While it does not strictly *need* to be included, it's good practice to specify it for cross-browser consistency.
 - `SameSite=Strict` continues to be a more restrictive option, for cookies that must only be sent when the user is already on the site. This is and remains a good security practice for cookies that are part of managing particularly sensitive access.
 - `SameSite=None` should continue to be sent for cross-browser consistency. However, Chrome's proposed change to phase out third-party cookies would result in those cookies no longer being sent as is in cross-site contexts.

The exception is cookies that are modified by either the [CHIPS](https://developer.chrome.com/docs/privacy-sandbox/chips/) or [First-Party Sets](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/) proposal. These allow for a subset of cross-site use cases. As these proposals are under active discussion, the final formats and functionality may change.
```

**Review by heyawhite — APPROVED** [review]
Will commit some small changes and merge

File: site/en/docs/privacy-sandbox/faq/index.md [hunk 5]
```diff
@@ -141,6 +141,20 @@ For more detailed information, see
...
+   be included, it's good practice to specify it for cross-browser consistency.
+- `SameSite=Strict` continues to be a more restrictive option, for cookies that
+   must only be sent when the user is already on the site. This is and remains
+   a good security practice for cookies that are part of managing particularly
+   sensitive access.
+- `SameSite=None` should continue to be sent for cross-browser consistency. However,
+   Chrome's proposed change to phase out third-party cookies would result in those
+   cookies no longer being sent as is in cross-site contexts.
+
+The exception is cookies that are modified by either the [CHIPS](https://developer.chrome.com/docs/privacy-sandbox/chips/) or [First-Party Sets](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/) proposal. These allow for a subset of cross-site use cases. As these proposals are under active discussion, the final formats and functionality may change.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/faq/index.md:156]:
```suggestion
The exception is cookies that are modified by either the
[CHIPS](/docs/privacy-sandbox/chips/) or
[First-Party Sets](/docs/privacy-sandbox/first-party-sets/) proposal.
These allow for a subset of cross-site use cases. As these proposals are
under active discussion, the final formats and functionality may change.
```

**Pull request merged**